config/import-control.xml

```diff
@@ -14,6 +14,13 @@
         <allow pkg="org.immutables.gson"/>
     </subpackage>
 
+    <subpackage name="extract">
+        <allow pkg="com.google.gson"/>
+        <allow class="com.sun.tools.javac.util.ServiceLoader"/>
+        <allow class="java.lang.reflect.Type"/>
+        <allow class="java.nio.charset.Charset"/>
```

Comment:
This shouldn't be needed since it is not imported in main code.
Please remove and confirm the others are needed.